Makefile

```diff
@@ -41,7 +41,7 @@ ifndef KODEBUG
 		$(if $(or $(DARWIN),$(WIN32),$(ANDROID),$(UBUNTUTOUCH),$(APPIMAGE)),,$(OUTPUT_DIR)/dropbear) \
 		$(if $(or $(KINDLE),$(KOBO)),$(OUTPUT_DIR)/sftp-server,) \
 		$(if $(or $(KINDLE),$(KOBO)),$(OUTPUT_DIR)/scp,) \
-		$(if $(or $(KINDLE),$(KOBO)),$(OUTPUT_DIR)/fbink,) \
+		$(if $(or $(KINDLE),$(KOBO),$(CERVANTES)),$(OUTPUT_DIR)/fbink,) \
```

Comment:
that's weird, is your tree updated? In master we actually build fbink and sftp-server for cervantes: https://github.com/koreader/koreader-base/blob/master/Makefile#L30-L32

Reply:
It was indeed building, that's the strip step on !DEBUG builds ;).

Reply:
Oh, cool. Could you strip sftp-server too for me? or do I need to open a new PR?